docs/specification/1.0.md

```diff
@@ -473,6 +473,7 @@ Field | Optionality | Type | Description
 ----- | ----- | ----- | --------
 `label` | REQUIRED | *string* | Human-readable label to display for this suggestion (e.g. the CDS Client might render this as the text on a button tied to this suggestion).
 `uuid` | OPTIONAL | *string* | Unique identifier, used for auditing and logging suggestions.
+`isRecommended` | OPTIONAL | *boolean* | Allows a service to indicate that a specific suggestion is recommended from all the available suggestions on the card. CDS Hooks clients may choose to influence their UI based on this value, such as pre-selecting, or highlighting recommended suggestions. Multiple suggestions MAY be recommended, if `card.selectionBehavior` is `any`. A CDS Service MUST only recommend one (or more) suggestions if that service expects the suggestion(s) to be accepted a majority of the time.
```

Comment:
```suggestion
`isRecommended` | OPTIONAL | *boolean* | Allows a service to indicate that a specific suggestion is recommended from all the available suggestions on the card. CDS Hooks clients may choose to influence their UI based on this value, such as pre-selecting, or highlighting recommended suggestions. Multiple suggestions MAY be recommended, if `card.selectionBehavior` is `any`. 
```

I'd recommend cutting out the following sentence: 

> A CDS Service nd one (or more) suggestions if that service expects the suggestion(s) to be accepted a majority of the time.

... because I'm not sure it's the right rubric, and I don't know how a service should derive "expectations." In other words, the combination of hard conformance language ("MUST") with a squishy assessment doesn't sit quite right with me.